jaqpotpy/jaqpot.py

```diff
@@ -126,13 +126,18 @@ def deploy_sklearn_model(self, model, name, description, visibility):
                     feature_type=feature_i["featureType"],
                     possible_values=feature_i["possible_values"]
                     if "possible_values" in feature_i
+                    and (
```

Comment:
what is this for here?

Reply:
it is to pass only the selcteded independent features into the database after feature selection. 
for example if i have a dataframe of X1,X2 and i want to select only X1 in training, then X2 doesnt need to be in the UI.

Reply:
But why hardcode the SMILES key?

Reply:
What if the user doesn't want to keep the SMILES string? You're forcing them to keep it with this hardcoded value, I think it's better to let them know with an error when they're uploading the model that they need to add the SMILES key if they want their featurizer to work, not hardcoded it silently